client/client-multi/client-objective-c/Kaa/avro/gen/EndpointGen.h

```diff
@@ -24,6 +24,7 @@
 #import "KAAUnion.h"
 
 
+
```

Comment:
Apparently this line was removed right after last file generation. I'd not consider that as even minor change.